medicaltorch/transforms.py

```diff
@@ -421,6 +427,29 @@ def __call__(self, sample):
         sample.update(rdict)
         return sample
 
+    def undo_transform(self, sample):
+        rdict = {}
+
+        angle = - sample['input_metadata']['randomRotation']
```

Comment:
That would work only with a single input, no?

Reply:
Yep, you are likely to be true.
I thought the same thing.. found a reason to do this way... but can't find it anymore.. so I corrected this accordingly :D